proto/src/lib.rs

```diff
@@ -366,17 +368,16 @@ mod tests {
 
     #[test]
     fn test_modify_from_raw() {
-        use lber::Consumer;
         use std::convert::TryFrom;
 
         let mut parser = lber::parse::Parser::new();
-        let (_size, msg) = match *parser.handle(lber::Input::Element(&[
+        let (_rem, msg) = match parser.parse(&[
             48, 69, 2, 1, 2, 102, 64, 4, 39, 117, 105, 100, 61, 98, 106, 101, 110, 115, 101, 110,
             44, 111, 117, 61, 80, 101, 111, 112, 108, 101, 44, 100, 99, 61, 101, 120, 97, 109, 112,
             108, 101, 44, 100, 99, 61, 99, 111, 109, 48, 21, 48, 19, 10, 1, 2, 48, 14, 4, 2, 115,
             110, 49, 8, 4, 6, 77, 111, 114, 114, 105, 115,
-        ])) {
-            lber::ConsumerState::Done(size, ref msg) => (size, msg),
+        ]) {
+            Ok(v) => v,
```

Comment:
Here and below: `.unwrap()`, or `.expect(...)`

Reply:
Good points, you can tell this is some of my older code :)